app/views/fragments/checkout/fieldsPersonal.scala.html

```diff
@@ -1,57 +1,62 @@
 @import com.gu.i18n.Title
 @import model.PersonalData
 
-@(form: Option[PersonalData])
+@(form: Option[PersonalData], productSupportsGifting: Boolean)
 
 @signedInAttrs = @{ if (form.isDefined) "required readonly" else "" }
 
 @* ===== Name ===== *@
 <div class="form-field js-checkout-title">
-    <label class="label" for="title">Title</label>
```

Comment:
This was pulled out into its own template used by the gift (delivery) and the personal sections.